File: test.store/Test.Store.Entities/Order.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Test.Store.Entities
{
    public class Order
    {
        public int custid { get; set; }
        public int empid { get; set; }
        public DateTime orderdate { get; set; }
        public DateTime  requireddate { get; set; }
        public DateTime shippeddate { get; set; }
        public int shipperid { get; set; }
        public decimal freight { get; set; }
        public string  shipname { get; set; }
        public string shipaddress { get; set; }
        public string shipcity { get; set; }
        public string shipregion { get; set; }
        public string shippostalcode { get; set; }
        public string shipcountry { get; set; }
        public OrderDetail orderdetail { get; set; }
  
    }
}

File: test.store-front/src/app/models/order.ts
import { OrderDetail } from "./orderDetail";

export interface Order
{
custid:number,
empid:number,
orderdate:Date,
requireddate :Date,
shippeddate:Date,
shipperid:number,
freight:number,
shipname:string,
shipaddress:string,
shipcity:string,
shipregion:string,
shippostalcode:string,
shipcountry:string,
orderdetail:OrderDetail
}